src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -60,8 +60,57 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
     }
   });
 
+  // verify wbs element exists
+  const wbsElement = await prisma.wBS_Element.findUnique({
+    where: {
+      wbsElementId: updated.wbsElementId
+    },
+    include: {
+      workPackage: true
+    }
+  });
+
+  if (wbsElement === null)
+    return res
+      .status(404)
+      .json({ message: `wbs element with id #${updated.wbsElementId} not found` });
+
+  const progress: number | undefined = wbsElement.workPackage?.progress;
+
+  if (updated.accepted && foundCR.type === CR_Type.STAGE_GATE) {
+    prisma.work_Package.update({
```

Comment:
i spent like 10 minutes debugging before realizing it doesn't work because you didn't put await here lmao